rhine/rhine.cabal

```diff
@@ -106,7 +106,8 @@ library
   -- Directories containing source files.
   hs-source-dirs:      src
 
-  ghc-options:         -Wall
+  ghc-options:         -Wall -Wno-unused-imports -Wno-unticked-promoted-constructors
```

Comment:
Is it possible to put them on separate lines? (Makes it easier for git to merge if there are edits in different places in this section)

Reply:
Done.